src/Compilers/Core/Portable/Compilation/Compilation.cs

```diff
@@ -973,7 +973,7 @@ protected static ImmutableArray<string> CheckTupleElementNames(int cardinality,
                 {
                     if (elementNames[i] == "")
                     {
-                        //throw new ArgumentException(CodeAnalysisResources.TupleElementNameEmpty, $"{nameof(elementNames)}[{i}]");
+                        throw new ArgumentException(CodeAnalysisResources.TupleElementNameEmpty, $"{nameof(elementNames)}[{i}]");
```

Comment:
this check is really bit strange. so null or "<space>+" is okay but "" not okay?

Reply:
ping @jcouv